app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,25 @@
+class FoodsController < ApplicationController
+  def create
+    @food = Food.new(food_params)
+
+    if @food.save
+      redirect_to root_path, notice: 'Food successfully added'
+    else
+      render :new, notice: 'Failed'
+    end
+  end
+
+  def new; end
+
+  def destroy
+    @food = Food.find(params[:id])
+    @food.destroy
+    # redirect_to foods_path, notice: 'Food deleted successfully'
+  end
+
+  private
+
+  def food_params
```

Comment:
@mckent05 We have to follow the wireframes of the 3 members,  
![image](https://github.com/BiancaMe/recipe-app/assets/37249375/f3ee74b6-e3c0-4e87-9255-7a20df864ebb)
We reported that our partner left the project, and Student Success said that we could finish the tasks of the member who left the project to submit.

Reply:
- After having discussions with a SCR, I have been informed to overlook this requirement since they started off as a team of 3 and implemented the requirements